### PR TITLE
feat(server/client): service quick reload

### PR DESCRIPTION
**Pull request by muink**
Allow manual quick reload of hp service

### CODE REVIEW

File: htdocs/luci-static/resources/view/homeproxy/client.js [hunk 1]
```diff
@@ -340,6 +341,13 @@ return view.extend({
 		}
 		so.default = 'nil';
 		so.rmempty = false;
+
+		so = ss.option(form.Button, '_reload_client', _('Quick Reload'));
```
Comment (1715173329) [inline htdocs/luci-static/resources/view/homeproxy/client.js:345]: I don't see any sense of this change.

Comment (muink) [reply htdocs/luci-static/resources/view/homeproxy/client.js:345]: emmm... I can probably list a few points.

1. Faster than `Default outbound: proxy --> nil --> SAVE --> proxy --> APPLY`
2. This is faster when hp stops due to itself or other reasons

Comment (1715173329) [reply htdocs/luci-static/resources/view/homeproxy/client.js:345]: 1. you have to edit uci configuration to stop the service if use `reload` method.
2. procd should automatically restart the service

Comment (muink) [reply htdocs/luci-static/resources/view/homeproxy/client.js:345]: 1. But stop_service does not check the value of `outbound_node`, it just terminates the service completely.
2. procd sometimes fails to respawn, and I have set the watchdog to stop the hp service when the proxy node is offline for a long time

Comment (1715173329) [reply htdocs/luci-static/resources/view/homeproxy/client.js:345]: 1. that will not _stop_ the service anyway, it's just restart. manually restart can be done via `system` -> `startup`.
2. then you should check logs, why it fails. procd will only stop respawn if the service crashed 5 times in one hour.

**Pull request closed without merging**